doc/doc-site/docs/off-chain/dss/ci-cd.md

```diff
@@ -142,3 +142,19 @@ DSS_SOURCE_REV=dss-v1.5.0
 TERRAFORM_PROJECT_REV=dss-v1.5.0
 source scripts/hot-upgrade.sh $DSS_SOURCE_REV $TERRAFORM_PROJECT_REV
 ```
+
+## Suspend/resume runner
+
+In order to avoid getting charged for a mostly unused service, you can suspend the runner when you're not using it, and resume it before hot upgrading for example.
+
+To do so, you can use `scripts/suspend-runner.sh` like so:
+
+```bash
+source scripts/suspend-runner.sh
+```
+
+and `scripts/resume-runner.sh` like so:
+
+```bash
+source scripts/resume-runner.sh
+```
```

Comment:
How about a note down here saying that if you resume the runner, give it a minute or two to start up before you re-engage with the project?

Reply:
Oof, it looks like you might then need to make something like `get-runner-startup-logs.sh` but for `get-runner-resumption-logs.sh` which accepts an argument for the project to check in